includes/api/class-wc-admin-rest-onboarding-levels-controller.php

```diff
@@ -21,7 +21,7 @@ class WC_Admin_REST_Onboarding_Levels_Controller extends WC_REST_Data_Controller
 	 *
 	 * @var string
 	 */
-	protected $namespace = 'wc/v4';
+	protected $namespace = 'wc-internal/v1';
```

Comment:
@justinshreve I know we discussed this on slack but perhaps `wc-admin` would be more appropriate which follows `wc-blocks` lead - both are internal APIs (we just need to work out how to lock them all down).

Reply:
Updated to `wc-admin/v1`.